Samples/dir.targets

```diff
@@ -5,10 +5,24 @@
   </PropertyGroup>
   
   <Import Project="..\dir.targets" />
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets"/>
```

Comment:
When using Microsoft.CSharp.targets instead of Microsoft.Portable.CSharp.targets, I get the following warning:

> C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1097,5): warning MSB3644: The reference assemblies for framework "DNXCore,Version=v5.0" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [C:\git\msbuild\src\Framework\Microsoft.Build.Framework.csproj]

I think we'll need a NuGet package that provides the portable targets for us.  For now, can you make it so that when building on Windows and `NetCoreBuild` is true, that it will import `Microsoft.Portable.CSharp.targets` instead of `Microsoft.CSharp.targets`?